docs/how_to/traefik/traefik_handbook.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: traefik_handbook
+title: Traefik Handbook
+sidebar_label: Traefik Handbook
+---
+
+import SourceCodeBlock from "./../../../src/components/SourceCodeBlock";
+
+## Add Basic Authorization
+
+:::caution
+This feature must not be used in production environments.
+
+The basic auth must be always replaced with safer authentication methods also in other environments.
+:::
+
+To add basic authentication on some host there are few steps to follow:
+
+### Create the Middleware
+First of all you need to create a new Middleware. This is a yaml configuration.
+
+Based on the Console configuration there are two options where to put the new file:
+1. `configuration/{{ENV}}/`
+2. `overlays/{{ENV}}/` in case of Kustomize Configuration
+
+The file can be named as desired, in this example we will refer it as `default.basicauth.yml`
+
+```yaml
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: default-basicauth
+  labels:
+    app.kubernetes.io/instance: "ingress-controller"
+spec:
+  basicAuth:
+    secret: authsecret
+```
+
+### Create a new secret
+
+Create a new secret with the basic auth credentials. The [Traefik documentation](https://doc.traefik.io/traefik/middlewares/http/basicauth/#general) suggest to use the `htpasswd` command to evaluate the secret.
+
+Example with username and password:
+```
+echo $(htpasswd -nb username password)
+```
+
+The output of the above command is the value of the secret to create.
+
+The secret needs to be created through one of the [supported environments variable provider](https://docs.mia-platform.eu/docs/development_suite/set-up-infrastructure/env-var).
+In this hands-on we will refer to the secret as `BASIC_AUTH_USERS`.
+
+#### Generate the secret with mlp
+We also need to edit the [mlp](https://docs.mia-platform.eu/docs/runtime_suite_tools/mlp/generate) configuration to interpolate the secret created in the previous section.
+
+In the `mlp.yml` file add this configuration:
+```
+  - name: "authsecret"
+    when: "always"
+    data:
+      - key: "users"
+        value: "{{BASIC_AUTH_USERS}}"
+        from: literal
+```
+
+### Add the Middleware to the Ingress
+The last step is to introduce the Middleware in the IngressRoute configuration. The config can be found inside:
+- `configuration/{{env}}/default.ingressroute.yml`, or
+- `overlays/{{ENV}}/default.ingressroute.yml`
+
+Add the middleware `default-basicauth` in the host where you want to setup the basic auth.
+Example:
+```
+routes:
+    - match: Host(`my-hostname.eu`)
+      middlewares:
+        ...
+        - name: "default-basicauth"
+```
+### Deploy
```

Comment:
It's not a tutorial. I would remove the deploy chapter.

Reply:
added closing sentence